client/app/components/project/package-list.hbs

```diff
@@ -1,19 +1,17 @@
-<h2>{{@packageName}}</h2>
 <ol class="no-bullet">
   <li>
-    <!-- TODO update all ifs with real logic -->
     <a href="#">
-      {{#if (eq status "revision required")}}
+      {{#if (eq @package.statuscode (optionset 'package' 'status' 'code' "Reviewed - Revisions Required"))}}
         <FaIcon @icon="exclamation-triangle" class="text-gold" />
       {{else}}
         <FaIcon
-          @icon={{if (eq status "package prep") 'edit' 'check'}}
-          class={{if (eq status "package prep") '' 'text-green-dark'}}
+          @icon={{if (eq @package.statuscode (optionset 'package' 'status' 'code' "Package Preparation")) 'edit' 'check'}}
```

Comment:
Yay! thanks for using optionset. 
The way you've written here works. As the last parameter, you're passing in the "human readable" label, and that is valid code.

However it's better to use the identifier `PACKAGE_PREPARATION`. This makes it slightly less error prone, and clear that it is a constant. 

e.g. 
```
(optionset 'package' 'status' 'code' 'PACKAGE_PREPARATION')
```

Reply:
good to know, will do!